clang/lib/AST/DeclBase.cpp

```diff
@@ -709,7 +711,7 @@ static AvailabilityResult CheckAvailability(ASTContext &Context,
             << EnvName << HintMessage;
```

Comment:
Do you want to avoid the extra space here?

Reply:
no 😅